src/csharp/Tensor.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.OnnxRuntimeGenAI
+{
+    public enum ElementType : long
+    {
+        undefined,
+        float32,
+        uint8,
+        int8,
+        uint16,
+        int16,
+        int32,
+        int64,
+        string_t,
+        bool_t,
+        float16,
+        float64,
+        uint32,
+        uint64,
+    }
+
+    public class Tensor : IDisposable
```

Comment:
Documentation.

Reply:
We plan on having it here once it stabilizes: https://onnxruntime.ai/docs/genai/api/csharp.html